src/test/java/EdsonAnawayaTests/EdsonAnawayaTest.java

```diff
@@ -0,0 +1,48 @@
+package EdsonAnawayaTests;
+
+import EdsonAnawayaTests.Entity.Project;
+import io.restassured.response.Response;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import static EdsonAnawayaTests.Entity.TestStr.*;
+
+public class EdsonAnawayaTest {
+
+    @BeforeClass
+    public void createRequestSpecification() {
+        configuration("https://api.todoist.com", "/rest/v1");
+    }
+
+    @Test
+    public void testGetProjects() {
+        authorization();
+        Response response = get("/projects");
+        int expected = 200;
```

Comment:
there are many libraries that have the status codes. you don't need to create variables